src/pages/ListSwaps/SwapRow/index.tsx

```diff
@@ -0,0 +1,3 @@
+import SwapRow from "./SwapRow";
+
+export default SwapRow;
```

Comment:
:question: Why is this extra layer of indirection needed?

Reply:
Makes the imports from the outside unaware that the component is represented in a folder rather a single file.